### DISCUSSION TITLE
LATEX expressions support in README.md ?

### DISCUSSION THREAD

**Discussion by LPerdu** [community/community#8001] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi,

I believe that having GitHub rendering LATEX expressions in README.md files would be a really nice feature to add, especially for the scientific community.

It would be much cleaner than all the current workarounds such as including these expressions as images.

I think that GitLab is already supporting LATEX in its md files.

Have you planned to consider it someday ?

Cheers

**Comment by AnimeAllstar** [comment]
GitLab has a much more sophisticated feature set than GitHub. It can even generate flowcharts using [mermaid](https://about.gitlab.com/handbook/tools-and-tips/mermaid/). Hopefully, GitHub catches up soon. Latex in README.md files would be super useful.

Unrelated Edit: seems like GitHub is working on [mermaid support](https://github.com/github/roadmap/issues/372#issue-1100526417).

**Comment by davidstaheli** [comment, marked as answer]
Hi @LPerdu and @AnimeAllstar. Mermaid support was [released today](https://github.blog/2022-02-14-include-diagrams-markdown-files-mermaid/). It's definitely important for GitHub to support LaTeX expressions. We're in the middle of investigation that will determine how soon we can support it.
